eks-ebs-enable/README.md

```diff
@@ -0,0 +1,169 @@
+
+### Option 2 - permissions just for this tool
+
+Create a new IAM policy with the following permissions:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "iam:GetRole",
+                "eks:DescribeAddonConfiguration",
+                "eks:UpdateAddon",
+                "eks:ListAddons",
+                "iam:CreateRole",
+                "iam:AttachRolePolicy",
+                "eks:DescribeAddon",
+                "iam:CreateOpenIDConnectProvider",
+                "iam:PassRole",
+                "eks:DescribeIdentityProviderConfig",
+                "eks:DeleteAddon",
+                "iam:ListOpenIDConnectProviders",
+                "iam:ListAttachedRolePolicies",
+                "eks:CreateAddon",
+                "eks:DescribeCluster",
+                "eks:DescribeAddonVersions",
+                "sts:AssumeRoleWithWebIdentity",
+                "eks:AssociateIdentityProviderConfig",
+                "eks:ListIdentityProviderConfigs"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+Then create a new IAM user and attach the policy you just created. Then create a new access key and use the id and secret later.
+
+## Building
+
+The tool needs to be built. This done by doing the following:
+
+1. Clone the repo
+2. Change directory into this folder
+3. Run the following
+
+```shell
+make release-local
+```
+
+4. The **dist** folder contains:
+    1. archives for linux, macOS and windows
+    2. Linux packages in deb, rpm, apk format
+    3. SBOMs for the archives
+
+## Usage
+
+1. Create an API key in Rancher.The key can be scoped per cluster or with no scope. Its easier to have no scope as you can use the same API key for all cluster upgrades.
+   1. Note down the **Bearer Token** API key
+   2. Note down the API Endpoint
+2. Open a terminal
+3. Export environment variables for the key and endpoint
+
+```bash
+export RANCHER_TOKEN="<YOUR BEARER TOKEN FROM ABOVE>"
+export RANCHER_API="<YOUR RANCHER API ENDPOINT>"
+export KUBECONFIG="<YOUR RANCHER KUBECONFIG PATH>"
+```
+
+> If you are going to use the existing AWS credentials in Rancher (option 1 above) then you will need to use the `--kubeconfig` flag to point to the kubeconfig file for your Rancher clusters.
+
+1. Enable the addon for a cluster using one of the following commands depending on with iam permissions option you went with:
+    1. option 1 (augmenting existing permissions)
+
+    ```shell
+    eks-ebs-enable enable /
+        --endpoint $RANCHER_API \
+        --bearer-token $RANCHER_TOKEN \
+        --kubeconfig $KUBECONFIG \
+        --cluster richtest1 \
+        --debug
+    ```
+
+    2. option 2 (separate permissions)
+
+    ```shell
+    export AWS_ACCESS_KEY_ID="<YOU_KEY_ID>"
+    export AWS_SECRET_ACCESS_KEY="<YOUR_SECRET_KEY>"
```

Comment:
Maybe I'd not export these variables at all since it can interfere with `aws` and `eksctl` utils. Later I would just put  their values like:
```
  --access-key-id "<AWS_ACCESS_KEY_ID_VALUE>" \
  --access-key-secret "<AWS_ACCESS_KEY_SECRET_VALUE>"
```

Reply:
Changed to use strings and not env vars